cmd/juju/interact/pollster.go

```diff
@@ -393,7 +394,11 @@ func (p *Pollster) queryOneSchema(schema *jsonschema.Schema) (interface{}, error
 		// anything
 		a, err = p.Enter(schema.Singular)
 	case jsonschema.FormatURI:
```

Comment:
Hm, I don't think it needs to be. Is there anything wrong with broadening the scope of `VerifyURLS` to be just `Verify`?